broadway_api/callbacks/worker.py

```diff
@@ -34,37 +109,38 @@ def _handle_lost_worker_node(settings, worker):
 
     if not lost_run_id:
         logger.critical(
-            "worker {} went offline unexpectedly on {}".format(
-                worker.id, worker.hostname
+            "worker `{}` went offline unexpectedly on '{}' due to {}".format(
```

Comment:
Micro-Nit: Any reason we have used both ` and ' around {}. Maybe use one of them? Or is this intentional?

Reply:
Yeah I was trying to differentiate quotes back then --- but it doesn't seem meaningful now. I'll make them single quotes